esphome/components/waveshare_epaper/waveshare_epaper.h

```diff
@@ -26,7 +26,7 @@ class WaveshareEPaper : public PollingComponent,
 
   void update() override;
 
-  void fill(Color color) override;
+  virtual void fill(Color color) override;
```

Comment:
```suggestion
  void fill(Color color) override;
```

virtual is redundant here